.eslintrc.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  extends: [
+    'plugin:react/recommended',
+    'airbnb',
+  ],
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 12,
+    sourceType: 'module',
+  },
+  plugins: [
+    'react',
+  ],
+  rules: {
+    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
```

Comment:
Codacy has a fix for the issue: [Unnecessarily quoted property 'extensions' found.](https://app.codacy.com/gh/CBIIT/CCDC-WebPortal/pullRequest?prid=7925804)
```suggestion
    "react/jsx-filename-extension": [1, { extensions: [".js", ".jsx"] }],
```